pkg/github/client/client.go

```diff
@@ -51,3 +83,172 @@ func (client *Client) Query(ctx context.Context, q interface{}, variables map[st
 func (client *Client) ListWorkflows(ctx context.Context, owner, repo string, opts *googlegithub.ListOptions) (*googlegithub.Workflows, *googlegithub.Response, error) {
 	return client.restClient.Actions.ListWorkflows(ctx, owner, repo, opts)
 }
+
+// GetWorkflowUsage returns the workflow usage for a specific workflow.
+func (client *Client) GetWorkflowUsage(ctx context.Context, owner, repo, workflow string, timeRange backend.TimeRange) (models.WorkflowUsage, error) {
+	actors := make(map[string]struct{}, 0)
+
+	buildsPerDay := map[time.Weekday]uint64{
+		time.Monday:    0,
+		time.Tuesday:   0,
+		time.Wednesday: 0,
+		time.Thursday:  0,
+		time.Friday:    0,
+		time.Saturday:  0,
+		time.Sunday:    0,
+	}
+
+	conclusions := map[string]uint64{
+		"success":   0,
+		"failure":   0,
+		"cancelled": 0,
+		"skipped":   0,
+	}
+
+	usageDuration := time.Second * 0
```

Comment:
Won't this always evaluate to 0? Is there a reason it's not simply set to 0?

Reply:
If i just set it to zero the type of `usageDuration` won't be `time.Duration` and then down below when we try to add it to a value of type `time.Duration`, we'll get a type error:
```go
usageDuration += duration
```

I can change it to `var usageDuration time.Duration` if you think it makes it easier to understand

Reply:
Ah! That make sense! I think `var usageDuration time.Duration` would help the readability a bunch. :)

Reply:
Done